scripts/test-summary.sh

```
@@ -0,0 +1,54 @@
+echo ""
+echo "Unit Tests by Package:"
+for pkg in $(go list ./... | grep -v /vendor/); do
+    count=$(go test -list=. $pkg 2>/dev/null | grep -c "^Test" || true)
+    if [ -n "$count" ] && [ "$count" -gt 0 ]; then
+        echo "  $pkg: $count tests"
+    fi
+done
+
+echo ""
+echo "Integration Tests by Package:"
+for file in $(find . -name "*_integration_test.go" -not -path "./vendor/*" -not -path "./web/*"); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)